enchant.lic

```diff
@@ -81,13 +81,20 @@ class Enchant
 
     @equipment_manager.empty_hands
     get_item(@brazier) if @use_own_brazier
-    clean_brazier
-    bput("get my #{@item}", 'You get a')
+    
+    case bput("get my #{@item}", 'You get a','That is far too dangerous')
```

Comment:
Does this cover `an|a` item situations?

Reply:
Noticed there was no space after the `a` so  i will answer my own question with yes :D